ui-kit.code-workspace

```diff
@@ -0,0 +1,123 @@
+{
+  "folders": [
+    {
+      "name": "✨ ui-kit",
+      "path": "./"
+    },
+    {
+      "name": "📦 Auth",
+      "path": "./packages/auth"
+    },
+    {
+      "name": "📦 Bueno",
+      "path": "./packages/bueno"
+    },
+    {
+      "name": "📦 Headless",
+      "path": "./packages/headless"
+    },
+    {
+      "name": "📦 Quantic",
+      "path": "./packages/quantic"
+    },
+    {
+      "name": "📦 Atomic",
+      "path": "./packages/atomic"
+    },
+    {
+      "name": "📦 Atomic for React",
+      "path": "./packages/atomic-react"
+    },
+    {
+      "name": "📦 Atomic for Angular",
+      "path": "./packages/atomic-angular"
+    },
+    {
+      "name": "📦 Atomic Hosted Page",
+      "path": "./packages/atomic-hosted-page"
+    },
+    {
+      "name": "📄 Atomic Storybook documentation",
+      "path": "./utils/atomic-storybook"
+    },
+    {
+      "name": "🚀 IIFE samples",
+      "path": "./packages/samples/iife"
+    },
+    {
+      "name": "🚀 Stencil samples",
+      "path": "./packages/samples/stencil"
+    },
+    {
+      "name": "🚀 React samples (Headless)",
+      "path": "./packages/samples/headless-react"
+    },
+    {
+      "name": "🚀 React samples (Atomic)",
+      "path": "./packages/samples/atomic-react"
+    },
+    {
+      "name": "🚀 Next.js samples (Atomic)",
+      "path": "./packages/samples/atomic-next"
+    },
+    {
+      "name": "🚀 Angular samples",
+      "path": "./packages/samples/angular"
+    },
+    {
+      "name": "🚀 Vue.js samples",
+      "path": "./packages/samples/vuejs"
+    },
+    {
+      "name": "⚙️ Release utilities",
+      "path": "./utils/release"
+    },
+    {
+      "name": "⚙️ NX Adapter",
+      "path": "./utils/nx-plugin"
+    }
+  ],
+  "settings": {
+    "jest.disabledWorkspaceFolders": [
+      "✨ ui-kit",
+      "📦 Atomic",
```

Comment:
Atomic's tests aren't compatible with the Jest test runner unfortunately, and I haven't found a way to override settings for a single project.